cspell-dictionary.txt

```diff
@@ -0,0 +1,348 @@
+# custom dictionary for the heroku cli repository
```

Comment:
This first pass for all words to ignore is vital. It will make future PR spell checks much more effective.